src/clj/game/cards/agendas.clj

```diff
@@ -251,6 +251,14 @@
     :interactive (req true)
     :effect (effect (move-zone :runner :discard :rfg))}
 
+   "City Works Project"
+   {:install-state :face-up
+    :access {:req (req installed)
+             :msg (msg "do " (+ 2 (:advance-counter (get-card state card) 0)) " meat damage")
```

Comment:
I'd be tempted to extract the 2 + advancements function to reduce the minor repetition.

Reply:
Something like this?

```clojure
   "City Works Project"
   (let [ability (+ 2 (:advance-counter (get-card state card) 0))]
     {:install-state :face-up
      :access {:req (req installed)
               :msg (msg "do " ability " meat damage")
               :delayed-completion true
               :effect (effect (damage eid :meat ability {:card card}))}})
```

Reply:
Exactly! I'd name it something like `meat-damage` or `damage-strength`.